FPM.ftd

```diff
@@ -22,6 +22,18 @@ implements: fifthtry.github.io/theme
 -- fpm.auto-import: fpm
 
 
+-- fpm.user-group readers: fpm_readers
```

Comment:
Let's not event have anything after colon. Readers is the name of the group. This is brilliant.